eng/pipelines/core-splitting.yml

```diff
@@ -0,0 +1,74 @@
+parameters:
+- name: OutputFolder
+  type: string
+  default: projects
+- name: numOfTestsPreJob
+  type: number
+  default: 20
+steps:
+  - pwsh: |
+      dotnet build /t:ProjectDependsOn ./eng/service.proj `
+        /p:ServiceDirectory=* /p:TestTargetDependency="Azure.Core" `
+        /p:IncludeSrc=false /p:IncludeStress=false `
+        /p:IncludePerf=false /p:RunApiCompat=false /p:InheritDocEnabled=false
+    displayName: Find all test dependencies
+
+  - pwsh: | 
+      $packages = Select-Xml -Path ./eng/projects.props -XPath '/Project/ItemGroup' | ForEach-Object { $_.Node.ProjectReference }
```

Comment:
Lets move these steps and the previous step into an independent ps script so we can run and test it locally.